type.proto

```diff
@@ -77,6 +77,14 @@ message Object {
 }
 
 message ProfilingResult {
+  message Characteristic {
+    oneof member_oneof {
+      uint64 size = 1;
+      float decimal = 2;
+      string text = 3;
+    }
```

Comment:
this is the definition of a characteristic: `std::variant<int64_t, double, std::string>`. So I think this does not match with this protobuf definitions